build_tools/third_party/tracy/CMakeLists.txt

```diff
@@ -27,38 +27,71 @@ else()
 
   # Deps slightly differ by platform but some are common.
   pkg_check_modules(TRACY_DEPS
-    tbb
     libzstd
```

Comment:
Might be able to use https://github.com/wolfpld/tracy/tree/master/zstd and https://github.com/wolfpld/tracy/blob/16405715a6e4932c2b67548f433498f1bd93b619/cmake/vendor.cmake#L104-L146 instead of an install for this too